src/Take.Blip.Builder/Actions/ExecuteScript/ExecuteScriptAction.cs

```diff
@@ -91,18 +91,19 @@ public override async Task ExecuteAsync(IContext context, ExecuteScriptSettings
         protected async Task<object[]> GetScriptArgumentsAsync(
             IContext context, ExecuteScriptSettings settings, CancellationToken cancellationToken)
         {
-            object[] arguments = null;
             if (settings.InputVariables != null && settings.InputVariables.Length > 0)
             {
-                arguments = new object[settings.InputVariables.Length];
+                //improve memory alocation for arguments object
+                var arguments = new object[settings.InputVariables.Length];
```

Comment:
I think this will have very little impact, although it's more correct IMO.

Reply:
It will, but this code was baddly writed